tests/FSharp.Compiler.Service.Tests/FsiTests.fs

```diff
@@ -14,21 +14,15 @@ type Sentinel () =
 module MyModule =
     let test(x: int) = ()
 
-[<CollectionDefinition("FsiTests", DisableParallelization = true)>]
```

Comment:
Do I get it right that these are working even without applying the new xUnit matters?

Reply:
Yes, collection definition is pure xUnit.